tests/whitebox/integration/pool/test_add.py

```diff
@@ -128,7 +129,38 @@ def testAddDataCache2(self):
         with self.assertRaises(StratisCliActionError) as context:
             RUNNER(self._MENU + [self._POOLNAME] + devices)
         cause = context.exception.__cause__
-        self.assertIsInstance(cause, StratisCliInUseError)
+        self.assertIsInstance(cause, StratisCliInUseOtherTierError)
+        self.assertNotEqual(str(cause), "")
+
+
+class AddDataTestCase2(SimTestCase):
+    """
+    Test adding data devices where the devices are already in the data tier
+    of another pool.
+    """
+
+    _POOLNAME = "deadpool"
+    _SECOND_POOLNAME = "deadpool2"
+    _MENU = ["--propagate", "pool", "add-data"]
+    _DEVICES = _DEVICE_STRATEGY()
+    _SECOND_DEVICES = _DEVICE_STRATEGY()
+
+    def setUp(self):
+        super().setUp()
+        command_line = ["pool", "create", self._POOLNAME] + self._DEVICES
+        RUNNER(command_line)
+        command_line = ["pool", "create", self._SECOND_POOLNAME] + self._SECOND_DEVICES
+        RUNNER(command_line)
+
+    def testAddData(self):
+        """
+        Test that adding the same devices to a different data tier fails.
```

Comment:
^a different data tier^a data tier belonging to a different pool          ?

Reply:
Resolved.